app/helpers/application_helper/button/ems_timeline.rb

```diff
@@ -9,7 +9,7 @@ def disabled?
 
   def calculate_properties
     super
-    self[:hidden] = true if skip?
+    self[:hidden] = true if visible?
```

Comment:
I guess I need to get rid of the `:hidden` attribute somehow... Any idea is welcome :laughing: